### DISCUSSION TITLE
change favicon

### DISCUSSION THREAD

**Discussion by rwehe** [jpanther/lynx#19] [Q&A] [answered]
Just wondering what the easiest way is to change the favicon. I've tried putting it in the static/, static/img and modified the header file (head.html) that I have some custom code in under config/partials/head.html however I can't seem to get it working.

Thanks!

**Comment by rwehe** [comment, marked as answer]
figured it out... just needed to have the correct filenames and placed in the root of the static folder

**Comment by alxgda** [comment]
Hello, I'm unable to change the favicon. I put a favicon.ico icon file in the /static directory but I still get the purple square one.

**Reply by jpanther** [reply]
This sounds like a browser caching issue. Where are you deploying your site?

**Reply by alxgda** [reply]
I'm deploying it to Netlify. I just put the favicon.ico icon file, maybe I should also add the png ones. I'll try.

**Reply by alxgda** [reply]
Hi! So I'm confirming that I needed to add all the favicon files (favicon-16x16.png...) and not just the favicon.ico! Now it's working. Cheers!
